.github/workflows/ci.yaml

```diff
@@ -53,6 +57,12 @@ jobs:
             RUBYGEM_OVERCOMMIT=${{ matrix.rubygem_overcommit }}
             RUBYGEM_MODULESYNC=${{ matrix.rubygem_modulesync }}
 
+      - name: Upload voxbox-${{ matrix.rubygem_puppet }}_${{ github.sha }}.tar
+        uses: actions/upload-artifact@v4
```

Comment:
why does the step exist? so the other job can acccess the build artifact?

Reply:
TLDR: yes

if you build a container in on job, you cannot use it in another, because the local registry gets cleaned. So im uploading it as an artifact and donwload it in the next again

Reply:
was thinking about this too, but then forgot 👍